Adyen/Helpers/UILabelHelpers.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+extension UILabel {
+    
+    /// Initializes UILabel with given `TextStyle`
+    /// Sets `translatesAutoresizingMaskIntoConstraints` to `false`
+    /// Sets `adjustsFontForContentSizeCategory` to `true`
+    /// - Parameter style: `TextStyle` to be applied
+    convenience public init(style: TextStyle) {
+        self.init()
+        translatesAutoresizingMaskIntoConstraints = false
+        adyen.apply(style)
+    }
+    
+}
+
+public extension AdyenScope where Base: UILabel {
+    
+    /// Applies given `TextStyle` to the UILabel
+    /// Sets `adjustsFontForContentSizeCategory` to `true`
+    /// - Parameter style: `TextStyle` to be applied
+    internal func apply(_ style: TextStyle) {
+        base.font = style.font
+        base.textColor = style.color
+        base.textAlignment = style.textAlignment
+        base.backgroundColor = style.backgroundColor
+        round(using: style.cornerRounding)
```

Comment:
better remove this 🤔

Reply:
why? 🤔

Reply:
as I mentioned in mattermost - `round` can be percent-based - it is calculated based on real sizes of the element. If element is not yet set (layoutSubviews never called) the size is `.zero` so will be the rounding..